src/main/resources/application.yaml

```diff
@@ -8,7 +8,7 @@ spring:
       enabled: true
   jpa:
     hibernate:
-      ddl-auto: validate
+      ddl-auto: update
```

Comment:
validate should stay as we want only to check during the application startup if the database physical structure matches logical model.